src/fem/fe_p.hh

```diff
@@ -354,7 +368,9 @@ FE_P<degree,dim,spacedim>::FE_P()
         unit_support_points.push_back(dof_distribution.unit_support_points[i]);
 
     order = degree;
-
+    
+    this->setup_components();
```

Comment:
??

Reply:
Added documentation of FiniteElement::setup_components().